dockerfiles/Samtools/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM bashell/alpine-bash:latest
+
+ARG version=1.9
+
+RUN apk update \
+ && apk upgrade \
```

Comment:
I know `apt-get upgrade` is discouraged b/c of cacheing issues. May be the case with `apk` too?

Reply:
It is only doing updates in a separate run that leads to caching issues. From [Dockers best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

```Always combine RUN apt-get update with apt-get install in the same RUN```

Reply:
Cool- I think `update` is discouraged, but not for that reason. `Avoid RUN apt-get upgrade and dist-upgrade, as many of the “essential” packages from the parent images cannot upgrade inside an unprivileged container. `

Reply:
failed to process the upgrade part! Read it as a standard update + install. will remove

Reply:
Sorry- that was my fault for mixing up the reasons!